src/app/(site)/employee/my_page/edit/_components/EditMyPageFormContent.tsx

```diff
@@ -0,0 +1,199 @@
+'use client'
+
+import { Button } from "@/app/_components/ui/button";
+import { Textarea } from "@/app/_components/ui/textarea";
+import type { EmployeeDetailResponse } from "@/usecase/getEmployeeDetail/EmployeeDetailDTO";
+import { useForm } from '@conform-to/react';
+import { parseWithZod } from "@conform-to/zod";
+import type { Occupation, WorkLocation } from "@prisma/client";
+import * as Avatar from "@radix-ui/react-avatar";
+import type { Session } from "next-auth"
+import Link from "next/link";
+import { useFormState } from "react-dom";
+import { updateEmployeeForMyPageAction } from "../_actions/updateEmployeeForMyPageAction";
+import { updateEmployeeSchema } from "../_schema/updateEmployeeSchema";
+
+// SearchEmployeeBox.tsxと共通化したい
+const HIRING_TYPE = [
+  {
+    id: 1,
+    value: "NEW_GRADUATE",
+    label: "新卒入社",
+  },
+  {
+    id: 2,
+    value: "MID_CAREER",
+    label: "中途入社",
+  },
+  {
+    id: 3,
+    value: "BOTH",
+    label: "両方",
+  },
+] as const;
+
+const MEETING_METHOD = [
+  {
+    id: 1,
+    value: "ONLINE",
+    label: "オンライン",
+  },
+  {
+    id: 2,
+    value: "OFFLINE",
+    label: "対面",
+  },
+  {
+    id: 3,
+    value: "BOTH",
+    label: "全て",
+  },
+] as const;
+
+export const EditMyPageFormContent = ({ employee, session, occupations, workLocations }:
+  { session: Session, employee: EmployeeDetailResponse, occupations: Occupation[], workLocations: WorkLocation[] }) => {
+  const [lastResult, action] = useFormState(updateEmployeeForMyPageAction, undefined);
```

Comment:
```suggestion
  const [lastResult, action] = useFormState<FormState, FormData>(updateEmployeeForMyPageAction, initialState);
```

  lastResultの型（FormState）を定義してBEからのエラーもハンドリングできるようにする。

おそらくconformのsubmission.reply()から返却される値と、UseCaseから返却される値の繋ぎ込みをしないといけないのでそこが大変そう